app/javascript/utilities/useDisableGrammarly.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * Disable grammarly if certain conditions are met
```

Comment:
```suggestion
 * Get the textarea attributes required to disable the Grammarly browser extension in Chrome browsers
 * This is a temporary "fix" to get around this issue: https://github.com/forem/forem/issues/13814
```

docs (blocking) - I think we should reference the issue here - I can see people wondering why this exists 😅